packages/replay/src/util/createReplayEnvelope.ts

```diff
@@ -1,18 +1,17 @@
 import { Envelope, Event } from '@sentry/types';
 import { createEnvelope } from '@sentry/utils';
 
-import { REPLAY_SDK_INFO } from '../constants';
-
 export function createReplayEnvelope(
   replayId: string,
   replayEvent: Event,
   payloadWithSequence: string | Uint8Array,
 ): Envelope {
+  const { name, version } = replayEvent.sdk || {};
   return createEnvelope(
     {
       event_id: replayId,
       sent_at: new Date().toISOString(),
-      sdk: REPLAY_SDK_INFO,
+      sdk: { name, version },
```

Comment:
l: Should we maybe just pass `sdk: replayEvent.sdk` here?

Reply:
For envelopes in the SDK, we only send name and version in the header: https://github.com/getsentry/sentry-javascript/blob/2591d2aa63bee3de0fa741cfd266e5e46ca2baa5/packages/core/src/envelope.ts#L16-L24

According to the [dev spec](https://develop.sentry.dev/sdk/envelopes/#envelope-headers), we can send the entire `sdk` object though, so we could change it... I think for now we should go with SDK consistency, WDYT? Keeps the envelope a little smaller if we don't pass the integrations array twice for instance

Reply:
ahh, ok, I see. Sorry, this was/is a bit confusing for me, what goes on the event vs. what goes on the envelope 😅